### ISSUE TITLE
Uploading ID on membership signup 

### ISSUE THREAD

**Issue by ddayan** [rchain/marketing#22]
Upload fails multiple times. 
Users don't have guidelines on which formats and file names are supported (e.g. no spaces allowed)

**Comment by pmoorman** [comment]
Unfortunately, this is a known bug for a loooong time already.

The problem is mostly that these issues live not on the `website` or the `member-site`, but in the KYC app. We can solve this the "real way" by fixing it there, or maybe find workarounds by providing at least a wiki page or some other resource with clear guidelines on what you need to do if your upload fails.

For now, I would suggest removing the website and member-site label, since we can't do it within that scope. 

As an aside: it would really be a good idea if we'd bring the onboarding & KYC process more in line with the actual website, but that's a much bigger project than just a bug fix

**Comment by JoshOrndorff** [comment]
I totally agree that we should bring the onboarding inline with the website. That's why I added that label.

@pmoorman where does the code for the kyc app live?

**Comment by ddayan** [comment]
I heard that there's a working group that is going to suggest changes to bring on boarding to the new site. This issue is more about the current site, let's not wait months until the website is updated to fix something that could be fixed in a day.

**Comment by Ojimadu** [comment]
I think it's a simple problem to fix and it's very frustrating for user. I noticed that most of the errors while uploading KYC details comes from having spaces in the name of the file.

**Comment by dckc** [comment]
@pmoorman writes:
> Unfortunately, this is a known bug for a loooong time already.

Yeah; at least as far back as the March 2 [executive committee](https://www.gov.rchain.coop/executive-committee) meeting.

@JoshOrndorff writes:
> ... @pmoorman where does the code for the kyc app live?

Latest news I can find is a [Jun 30 comment][kb] from @kitblake:
>  Ian said that @n10n has the keys for the member.rchain.coop subdomain. So I mailed Navneet butI haven't heard back from him either.

[kb]: https://github.com/rchain/bounties/issues/595#issuecomment-401557405

**Comment by pmoorman** [comment]
> I heard that there's a working group that is going to suggest changes to bring on boarding to the new site. 

@ddayan do you know which people that working group is composed of? Or at least someone to contact?

@JoshOrndorff what I think we could do as a first step to bring onboarding in line with the website, is provide a sort of "sales page" of sorts that people go to before they enter the KYC. That page could provide motivation (by telling people why it's a good idea to become a member), and also tell them how the process works, and provide help along the way.

Another thing that I'm thinking off that we could do, is provide a flow for people to get onboarded on Discord first, before they do KYC. If you do Discord first, then at least when they get stuck people will know where to contact someone to get help.

Anyway, those suggestions are maybe out-of-scope to the particular issue Dror is raising.

**Comment by kitblake** [comment]
The contact person for the Membership signup app is @ian-bloom. 

It's not a simple thing to fix and we should probably just live with it until the next gen app lands.

**Comment by JoshOrndorff** [comment]
This is the first I'm hearing of the next gen app. Are there any details or formal plans?

**Comment by kitblake** [comment]
Last I heard it was on Deanna's desk, but today, idk.

ChrisB from LifeID is keenly interested in providing a solution, maybe check status with him.
